client/src/main/java/org/eclipse/hono/client/device/amqp/internal/AmqpAdapterClientCommandConsumer.java

```diff
@@ -0,0 +1,177 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.client.device.amqp.internal;
```

Comment:
That's right. As mentioned above, I tried to pack the new classes into better-named packages (also to prepare for splitting the clients later). Unfortunately, I could not move some classes because of the visibility of the code used there. If we give higher priority to putting them together in the same package, then we will probably have to put everything back into the `client` package.

Reply:
Do you think it should be moved to `client.impl`?

Reply:
I think the current structure is OK. Probably packages will later be reorganized anyway when splitting up the hono client module.